tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -7336,6 +7339,38 @@ <h2>Show similarity to selected datapoint</h2>
           }
         },
 
+        plotInferenceEvolution: function() {
+          const example = this.selectedExampleAndInference;
+          if (!example || !example.inferences || example.inferences.length < 2) {
+            return;
+          }
+          const inferences = example.inferences;
+          const data = [];
+          for (let i=0; i<inferences.length; i++) {
+            const inference = inferences[i];
+            for (let modelInd=0; modelInd<inference.length; modelInd++) {
+              const modelInference = inference[modelInd];
+              for (let labelInd=0; labelInd<modelInference.length; labelInd++) {
+                const value = modelInference[labelInd].score;
```

Comment:
for binary classifiers, only display the line chart for the '1' class, just as we do in partial dependence plots. the mirrored plots for class 1 and class 0 can be confusing and cluttering otherwise.